microsoft/testsuites/xfstests/xfstesting.py

```diff
@@ -548,6 +568,9 @@ def _execute_xfstests(
             cwd=xfstests.get_xfstests_path(),
             timeout=self.TIME_OUT,
         )
+        xfstests.create_send_community_test_msg(
```

Comment:
This method should be called in the xfstest tool, not in test case level.

Reply:
update